incubator/neo4j/values.yaml

```diff
@@ -3,26 +3,25 @@
 # Declare name/value pairs to be passed into your templates.
 # name: value
 
-Name: "neo4j"
-Image: "neo4j"
-ImageTag: "3.2.2-enterprise"
-ImagePullPolicy: "Always"
-NumberOfCores: 3
-NumberOfReadReplicas: 0
-MinAvailable: "67%"
+name: "neo4j"
+image: "neo4j"
+imageTag: "3.2.2-enterprise"
+imagePullPolicy: "Always"
+numberOfCores: 3
+numberOfReadReplicas: 0
 Component: "neo4j"
-BoltPort: 7687
-HttpPort: 7474
-DiscoveryPort: 5000
-TxPort: 6000
-RaftPort: 7000
-Resources:
+boltPort: 7687
+httpPort: 7474
+discoveryPort: 5000
+txPort: 6000
+raftPort: 7000
+resources:
```

Comment:
We've started recommending not to specify defaults for resources. Also, requests and/or limits is missing.You may do it like so:

```yaml
resources: {}
# limits:
#   cpu: 100m
#   memory: 512Mi
# requests:
#   cpu: 100m
#   memory: 512Mi
```